terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh

```diff
@@ -6,13 +6,13 @@
 # Note: this script is highly dependent on a convention used by Terra and AllOfUs
 # applications to place notebooks in the following directory structure:
 #
-#   ~jupyter-user/<workspace-name>/notebook.ipynb
+#   ~jupyter-user/<workspace-name>/<mode>/notebook.ipynb
 #
 # It exploits the fact that the CWD of a launching notebook is named after the workspace.
 # If notebooks are ever launched from other directories, this script will break.
 
 # The workspace name is simply the CWD of the running notebook.
 PWD="$(pwd)"
-export WORKSPACE_NAME="$(basename "$PWD")"
+export WORKSPACE_NAME=`basename $(dirname $(pwd))`
```

Comment:
this fixes the `WORKSPACE_NAME` issue after welder rollout

Reply:
Could the comment in the code here be updated? It is no longer accurate:

```
# The workspace name is simply the CWD of the running notebook.
```

Should be something like:

```
# Notebooks are copied into a subdirectory, such as "edit" or "safe", which is the CWD.
# The parent directory is the name of the workspace.
```

Thanks!